include/zephyr/sys_clock.h

```diff
@@ -4,344 +4,9 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-/**
- * @file
- * @brief Variables needed for system clock
- *
- *
- * Declare variables used by both system timer device driver and kernel
- * components that use timer functionality.
- */
-
 #ifndef ZEPHYR_INCLUDE_SYS_CLOCK_H_
```

Comment:
Should we rename this to something else (i.e. `ZEPHYR_INCLUDE_SYSCLOCK_H_`) so that the `sys/clock.h` can take the ownership of `ZEPHYR_INCLUDE_SYS_CLOCK_H_` and be consistent with everything else inside the `include/zephyr/sys/` folder?